lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb

```diff
@@ -15,6 +15,130 @@ def air_loop_hvac_unoccupied_threshold
+      'ASHRAE 169-2013-0B',
+      'ASHRAE 169-2013-1B',
+      'ASHRAE 169-2013-2B',
+      'ASHRAE 169-2013-3B',
+      'ASHRAE 169-2013-3C',
+      'ASHRAE 169-2013-4B',
+      'ASHRAE 169-2013-4C',
+      'ASHRAE 169-2013-5B',
+      'ASHRAE 169-2013-5C',
+      'ASHRAE 169-2013-6B',
+      'ASHRAE 169-2013-7A',
+      'ASHRAE 169-2013-7B',
+      'ASHRAE 169-2013-8A',
+      'ASHRAE 169-2013-8B'
+      economizer_type = 'FixedDryBulb'
+      drybulb_limit_f = 75
+    when 'ASHRAE 169-2006-5A',
+      'ASHRAE 169-2006-6A',
+      'ASHRAE 169-2013-5A',
+      'ASHRAE 169-2013-6A',
+      economizer_type = 'FixedDryBulb'
+      drybulb_limit_f = 70
+    else
+      economizer_type = 'FixedDryBulb'
+      drybulb_limit_f = 65
+    end
+
+    return [economizer_type, drybulb_limit_f, enthalpy_limit_btu_per_lb, dewpoint_limit_f]
+  end
+
+  # Determine if an economizer is required per the PRM.
+  #
+  # @param air_loop_hvac [OpenStudio::Model::AirLoopHVAC] air loop
+  # @param climate_zone [String] ASHRAE climate zone, e.g. 'ASHRAE 169-2013-4A'
+  # @return [Bool] returns true if required, false if not
+  def air_loop_hvac_prm_baseline_economizer_required?(air_loop_hvac, climate_zone)
+    economizer_required = false
+    baseline_system_type = air_loop_hvac.additionalProperties.getFeatureAsString('baseline_system_type').get
+    # System type 3 through 8 and 11, 12 and 13
+    if ['SZ_AC', 'PSZ_AC', 'PVAV_Reheat', 'VAV_Reheat', 'SZ_VAV', 'PSZ_HP', 'SZ_CV', 'PSZ_HP', 'PVAV_PFP_Boxes', 'VAV_PFP_Boxes'].include? baseline_system_type
+      case climate_zone
+      when 'ASHRAE 169-2006-2B',
+        'ASHRAE 169-2006-3B',
+        'ASHRAE 169-2006-3C',
+        'ASHRAE 169-2006-4C',
+        'ASHRAE 169-2006-5B',
+        'ASHRAE 169-2006-5C',
+        'ASHRAE 169-2006-6B',
+        'ASHRAE 169-2006-7A',
+        'ASHRAE 169-2006-7B',
+        'ASHRAE 169-2006-8A',
+        'ASHRAE 169-2006-8B',
+        'ASHRAE 169-2013-2B',
+        'ASHRAE 169-2013-3B',
+        'ASHRAE 169-2013-3C',
+        'ASHRAE 169-2013-4C',
+        'ASHRAE 169-2013-5B',
+        'ASHRAE 169-2013-5C',
+        'ASHRAE 169-2013-6B',
+        'ASHRAE 169-2013-7A',
+        'ASHRAE 169-2013-7B',
+        'ASHRAE 169-2013-8A',
+        'ASHRAE 169-2013-8B'
+        economizer_required = true
+      end
+    end
+
+    # System type 3 and 4 in computer rooms are subject to exceptions
+    if baseline_system_type == 'PSZ_AC' || baseline_system_type == 'PSZ_HP'
+      if air_loop_hvac.additionalProperties.hasFeature('zone_group_type')
+        if air_loop_hvac.additionalProperties.getFeatureAsString('zone_group_type').get == 'computer_zones'
+          economizer_required = false
+        end
+      end
+    end
+
+    # Check user_data in the zones
+    # As discussed for now, if a zone under the air loop hvac system has economizer exception
+    # The HVAC economizer should not be required.
+    # Another ticket may need to add to create a separate HVAC air loop for
```

Comment:
I don't think we need to comment on this here. No change would be needed in this method if we apply the exceptions to the system group method. I've added this issue to a Jira ticket so we don't lose track of it.

Reply:
Updated: https://github.com/NREL/openstudio-standards/blob/economizer_exception_prm/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb#L31